### ISSUE TITLE
AMS: Template does not allow author to customize page margins

### ISSUE THREAD

**Issue by JamesxX** [typst/templates#35]
https://github.com/typst/templates/blob/01afc5cb135743517111269cb34b98dc44400b17/unequivocal-ams/lib.typ#L53-L67

**Comment by laurmaedje** [comment]
Isn't this all mandated by AMS anyway?

**Comment by zatkins-dev** [comment]
Yes, but some of us would like to use this as the base for general documents, like we would with the `amsart` document class in LaTeX. 

If you try to set the margins after the `#show: ams-article.with(...)`, then the document will start on the second page, which is not at all desirable.

**Comment by noatmeal** [comment]
I just came to the issues about to make this exact same complaint. This template should default to looking like an AMS article but be flexible in how one can style it.
